src/app/routes/admin-forms.server.routes.js

```diff
@@ -11,6 +11,7 @@ let publicForms = require('../../app/controllers/public-forms.server.controller'
 let auth = require('../../app/controllers/authentication.server.controller')
 let submissions = require('../../app/controllers/submissions.server.controller')
 const emailSubmissions = require('../../app/controllers/email-submissions.server.controller')
+const EmailSubmissionsMiddleware = require('../../app/modules/submission/email-submission/email-submission.middlewares')
```

Comment:
nit: EmailSubmissionsMiddleware**s**

Reply:
hahahaha p sure middleware is uncountable though